CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
+### Added
+
+- adjust config for negative delays added as `transform_negative_delays()`. The tools
+take a config that contains positive and negative delays in the controllers and adds the
+minimum value (can be negative) to all other channels to synchronize them in the set-up.
```

Comment:
```suggestion
- Adjust config for negative delays added as `transform_negative_delays()`. The tools
take a config that contains positive and negative delays in the controllers and adds the
minimum (negative) value to all other channels to synchronize them in the set-up.
```
Because if the minimum is not negative, it will not do that :)